spread.yaml

```diff
@@ -31,6 +31,9 @@ environment:
     DELTA_REF: 2.17
     DELTA_PREFIX: snapd-$DELTA_REF/
     SNAPD_PPA_VERSION: "$(HOST: echo $SPREAD_SNAPD_PPA_VERSION)"
+    http_proxy: "$(HOST: echo $http_proxy)"
```

Comment:
AFAIK most of the tools that take into account the http proxy env vars honor both the uppercase and the lowercase versions, if the suite passes, could we define the spread env vars as uppercase as the rest of them are?